scripts/linux/fv/gitlab_test_fv.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/bash -x
```

Comment:
Below is `-ex` why here only `-x`

Reply:
Just added comments about that into script. Also added set +e for allowing online for exit not0.
Thanks for review.